src/main/java/com/iota/iri/Iota.java

```diff
@@ -71,6 +75,18 @@
  */
 public class Iota {
     private static final Logger log = LoggerFactory.getLogger(Iota.class);
+    public static final HashMap<String, Class<? extends Persistable>> COLUMN_FAMILIES =
```

Comment:
This seems off compared to the rest of the class.

I would rather see 2 separate enums for this instead of a "random" map in the main class.
The main class is now imported on multiple places just for these 2 family maps.

Reply:
The main class should only be used to inject dependencies.
I definitely agree with you that this detail doesn't belong here. It should belong to the `Tangle` class that needs to define what column families do we use.

Reply:
Another problem is that I used `HashMap` instead of `Map`.

Also I didn't use an immutable type...
In java 9 we will get `Map.of()`
I think I will leave it as it is now, and push soon for an upgrade to Java 11 :-)